storybook/stories/components/tooltip/1_composed_list.story.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { select } from '@storybook/addon-knobs';
+import React from 'react';
+
+import { TooltipWrapper, CustomTooltip, TooltipHeader, TooltipTable, TooltipTableColumn } from '@elastic/charts';
+import { isDefined } from '@elastic/charts/src/utils/common';
+
+import { tableSimple, simple, long } from './data';
+import { TooltipShowcase } from './tooltip_showcase';
+
+export const Example = () => {
+  const dataSet = select(
+    'dataSet',
+    {
+      'Simple - table': 'tableSimple',
+      'Simple - list': 'simple',
+      'Long - list': 'long',
+    },
+    'simple',
+  );
+  const dataSets = {
+    tableSimple,
+    simple,
+    long,
+  };
+  const columns: TooltipTableColumn[] = [
+    {
+      id: 'label',
+      renderCell: ({ label }) => <span className="echTooltip__label">{label}</span>,
+      textAlign: 'left',
+    },
+    {
+      id: 'value',
+      renderCell: ({ formattedValue }) => (
+        <span className="echTooltip__value" dir="ltr">
+          {formattedValue}
+        </span>
+      ),
+      textAlign: 'right',
+    },
+    {
+      id: 'markValue',
+      hidden: (items) => items.every(({ markValue }) => !markValue),
+      renderCell: ({ markValue, formattedMarkValue }) =>
+        isDefined(markValue) ? <span className="echTooltip__markValue">&nbsp;({formattedMarkValue})</span> : null,
+    },
+  ];
+  const MyTooltip: CustomTooltip = ({ header, values }) => {
+    return (
+      <TooltipWrapper>
+        <TooltipHeader header={header} />
+        <TooltipTable columns={columns} items={values} />
+      </TooltipWrapper>
+    );
+  };
```

Comment:
What if we don't expose the wrapper, but always wrap the passed components into it internally?
This could simplify a bit the code into:
```suggestion
  const MyTooltip: CustomTooltip = ({ header, values }) => {
    return (
      <>
        <TooltipHeader header={header} />
        <TooltipTable columns={columns} items={values} />
      </>
    );
  };
```
and the consumer doesn't need to think about the wrapper anymore

Reply:
Yeah, the `TooltipWrapper` is mainly to add the sticky styles and UI. Removed in 0a9484c.

So are you thinking that we just always add the sticky/actions UI whenever someone defines actions? Or should there be an option to remove that within the `TooltipSpec`?

Reply:
I think we will automatically add the stick/action UI if we define tooltip actions

Reply:
Ok that works for me. I used removed the wrapper in 0a9484c then we can just add the actions to the internal `TooltipWrapper` when we need to.